src/js/vosta.js

```diff
@@ -0,0 +1,550 @@
+'use strict';
+
+var $ = require('elements'),
+	clamp = require('mout/math/clamp'),
+	easeInOutQuad = require('kubrick-js/easing/easeInOutQuad'),
+	prefix = require('kubrick-js/lib/vendorprefix')(),
+	windowWidth = window.innerWidth,
+	editLeft = document.querySelector('.retouch-wrap .edit-left'),
+	editRight = document.querySelector('.retouch-wrap .edit-right'),
+	icons = [
+		$('.icon-graduate, .icon-sail'),
+		$('.icon-calendar, .icon-island'),
+		$('.icon-cog, .icon-leaf'),
+		$('.icon-growth, .icon-users'),
+		$('.icon-briefcase, .icon-brains'),
+		$('.icon-temple, .icon-certificate'),
+		$('.icon-form, .icon-globe'),
+		$('.icon-pencil, .icon-pen')
+	],
+	lines = $('.line-1, .line-2, .line-3');
+
+require('./modules/menu');
+require('./modules/back-to-top');
+
+require('kubrick-js')([
+	{
+		duration: '100%',
+		easing: 'linear',
+		actors: [
+			{
+				element: '.mood',
+				translateY: '-100%'
+			},
+			{
+				element: '.mood-sky',
+				translateY: '100%',
+				scale: 1.2
+			},
+			{
+				element: '.mood-waves',
+				translateY: '100%',
+				scale: 1.2
+			},
+			{
+				element: '.mood-ship',
+				scale: 1.2
+			},
+			{
+				element: '.case-summary',
+				translateY: '100%'
+			},
+			{
+				element: '.arrow-wrap',
+				translateY: '100%'
+			},
+			{
+				element: '.client-ship-back',
+				translateY: '-165%'
+			},
+			{
+				element: '.client-ship-front',
+				translateY: '-155%'
+			},
+			{
+				element: '.desc-client',
+				translateY: '-145%'
+			}
+		]
+	},
+	{
+		duration: '100%',
+		actors: [
+			{
+				element: '.client-ship-back',
+				translateY: ['-165%', '-250%']
+			},
+			{
+				element: '.client-ship-front',
+				translateX: '-10%',
+				translateY: ['-155%', '-260%']
+			},
+			{
+				element: '.desc-client',
+				translateY: ['-145%', '-250%']
+			},
+			{
+				element: '.grey-case',
+				translateY: '-100%'
+			}
+		]
+	},
+	{
+		duration: '100%',
+		actors: [
+			{
+				element: '.grey-case',
+				translateY: ['-100%','-250%']
+			},
+			{
+				element: '.cards',
+				translateY: '-100%'
+			},
+			{
+				element: '.cards-cards',
+				translateY: ['0','-10%']
+			},
+			{
+				element: '.cards-phone',
+				translateX: ['-95%','5%'],
+				translateY: ['95%','-5%']
+			}
+		]
+	},
+	{
+		duration: '100%',
+		actors: [
+			{
+				element: '.cards',
+				translateY: ['-100%','-300%']
+			},
+			{
+				element: '.cards-cards',
+				translateY: ['-10%','-20%']
+			},
+			{
+				element: '.cards-phone',
+				translateX: ['5%', '105%'],
+				translateY: ['-5%', '-105%']
+			}
+		]
+	},
+	/* !!! done until here !!! */
```

Comment:
And under here, too, along with any variables from the top that are not used (and probably the icon positioning function)